tools/observability/README.md

```diff
@@ -1 +1,35 @@
-[See docs here](https://docs.determined.ai)
\ No newline at end of file
+[See docs here](https://docs.determined.ai)
+
+# Metrics
+Determined does not produce metrics, rather it uses existing tools to report information. Please see source documentation for details on specific metrics.
```

Comment:
Did we ever decide on removing det state metrics? 

I know that seems to report a few metrics? Do you think we need to document those?

https://github.com/determined-ai/determined/blob/43289e9afcf780ec8b0bc7695b450408d391d2bf/master/internal/prom/det_state_metrics.go#L17

Reply:
We didn't decide whether we were going to remove det state metrics or not ... but we did decide that we weren't going to document them or include them in any dashboards, etc going forward. For now, I think the philosophy is "don't stop customers from using them if they already are, but don't encourage anyone to start using them either".

Reply:
Nope! This came up during planning; we are specifically not going to document the det state metrics to pave the away for future deprecation.